docs/development/testguide.rst

```diff
@@ -36,7 +36,9 @@ Astropy, and will also be of general use to other packages.
 The ``pytest-astropy`` package is compatible with with the 2.0 series of
 Astropy, starting with 2.0.3. The plugins provided by ``pytest-astropy`` will
 be used if they are installed. If they are not installed, Astropy will fall
-back to using internally packaged versions of the plugins.
+back to using internally packaged versions of the plugins. (Specifically, it
+will use the versions of the plugins that correspond to
```

Comment:
hmm, I think it's better not to mention the version number here, as it most probably will be forgotten to be updated. For similar things we only mention the versions in the changelog (or when there is a minimum requirement, in the install page)

Reply:
Sounds good. That occurred to me too, but I updated it against my better judgment anyway :man_facepalming:.